nucypher/blockchain/eth/sol/source/contracts/StakingEscrow.sol

```diff
@@ -109,25 +109,113 @@ contract StakingEscrow is Issuer, IERC900History {
     using Snapshot for uint128[];
     using SafeERC20 for NuCypherToken;
 
+    /**
+    * @notice Signals that tokens were deposited
+    * @param staker Staker address
+    * @param value Amount deposited (in NuNits)
+    * @param periods Number of periods tokens will be locked
+    */
     event Deposited(address indexed staker, uint256 value, uint16 periods);
+
+    /**
+    * @notice Signals that tokens were stake locked
+    * @param staker Staker address
+    * @param value Amount locked (in NuNits)
+    * @param firstPeriod Starting lock period
+    * @param periods Number of periods tokens will be locked
+    */
     event Locked(address indexed staker, uint256 value, uint16 firstPeriod, uint16 periods);
+
+    /**
+    * @notice Signals that a sub-stake was divided
+    * @param staker Staker address
+    * @param oldValue Old sub-stake value (in NuNits)
+    * @param lastPeriod Final locked period of old sub-stake
+    * @param newValue New sub-stake value (in NuNits)
+    * @param periods Number of periods to extend sub-stake
+    */
     event Divided(
         address indexed staker,
         uint256 oldValue,
         uint16 lastPeriod,
         uint256 newValue,
         uint16 periods
     );
+
+    /**
+    * @notice Signals that two sub-stakes were merged
+    * @param staker Staker address
+    * @param value1 Value of first sub-stake (in NuNits)
+    * @param value2 Value of second sub-stake (in NuNits)
+    * @param lastPeriod Final locked period of merged sub-stake
+    */
     event Merged(address indexed staker, uint256 value1, uint256 value2, uint16 lastPeriod);
+
+    /**
+    * @notice Signals that a sub-stakes was prolonged
+    * @param staker Staker address
+    * @param value Value of sub-stake
+    * @param lastPeriod Final locked period of sub-stake
+    * @param periods Number of periods sub-stake was extended
+    */
     event Prolonged(address indexed staker, uint256 value, uint16 lastPeriod, uint16 periods);
+
+    /**
+    * @notice Signals that tokens were withdrawn to the staker
```

Comment:
Small nitpick: in some events `staker` starts from lowercase letter in other places - from capital

Reply:
✔️